### ISSUE TITLE
How to curate results on pbench-server ?

### ISSUE THREAD

**Issue by Deepthidharwar** [distributed-system-analysis/pbench#478]
Once we move the results, using pbench-move-results, it's housed in the server.
Is there any way to delete certain results from the server ? 
Currently, I have had so many runs and I want to clean up unwanted logs on the system.

**Comment by ndokos** [comment]
There is no way to delete results from the server currently. However, assuming you are looking at your results through the http:.//server/results hierarchy, you can reorganize your results by using `pbench-edit-prefix` (or by using `--prefix=foo` as an option to `pbench-move-results` if you are prescient enough to know where they should go in the first place). That should resolve the visual clutter at least.

**Comment by Deepthidharwar** [comment]
Sure, that would help me make sort and make it more readable. 
In the long run, I have had so many results on the server and eventual it would end up in a clutter.
Would it be good to provide 'delete' option ? Is it on the cards ?

**Comment by ndokos** [comment]
Not really. If you want to clean up, just do `pbench-edit-prefix --prefix=to-delete <result>` for each result you want to get rid of. We'll probably only start deleting results if we are under a disk space crunch. We have talked however of marking some results "to-keep" and then aging out results that are not so marked. That might still happen at some point.

**Comment by Deepthidharwar** [comment]
Sure! Sounds good.

Thank you so much.

**Comment by portante** [comment]
Let's consider adding this to the documentation.

**Comment by portante** [comment]
FYI, we are planning on deprecated the `pbench-edit-prefix` interface, see #1355; future work should focus these enhancements in the pbench dashboard.

**Comment by pravins** [comment]
With latest server, we can curate the results. 
1. users will have data ownership and can manage results on their own.
